specs/AgriFood/AgriCrop/schema.json

```diff
@@ -0,0 +1,110 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/AgriCrop/schema.json",
+    "title": "FIWARE - Agri Food",
+    "description": "This entity contains a harmonised description of a generic crop. This entity is primarily associated with the agricultural vertical and related IoT applications.",
+    "type": "object",
+    "properties": {
+	"id": {
+	    "type": "string",
+	    "format": "uri"
+	},
+	"type": {
+	    "type": "string",
+	    "enum": ["AgriCrop"],
+	    "description": "NGSI Entity Type"
+	},
+	"@context": {
+	    "type": "array",
+	    "items": [
+		{
+		    "type": "string",
+		    "enum": [
+			"https://forge.etsi.org/gitlab/NGSI-LD/NGSI-LD/raw/master/coreContext/ngsi-ld-core-context.json",
+			"https://raw.githubusercontent.com/GSMADeveloper/NGSI-LD-Entities/master/examples/Agri-Crop-context.jsonld"
+		    ]
+		}
+	    ]	     
+	},
+	"createdAt": {
+	    "type": "string",
+	    "format": "date-time"
+	},
+	"modifiedAt": {
+	    "type": "string",
+	    "format": "date-time"
+	},
+	"source": {
+	    "type": "string",
+	    "format": "url"
+	},
+	"dataProvider": {
+	    "type": "string",
+	    "format": "url"
+	},
+	"entityVersion": {
+	    "type": "number",
+	    "minimum": 2.0
+	},
+	"name": {
+	    "$ref": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/common-schema.json#/definitions/PropertyString"
+	},
+	"alternateName": {
+	    "$ref": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/common-schema.json#/definitions/PropertyString"
+	},
+	"description": {
+	    "$ref": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/common-schema.json#/definitions/PropertyString"
+	},
+	"agriSoil": {
+	    "$ref": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/common-schema.json#/definitions/Relationship"
+	},
+	"agriFertiliser": {
+	    "$ref": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/common-schema.json#/definitions/Relationship"
+	},
+	"agriPest": {
+	    "$ref": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/common-schema.json#/definitions/Relationship"
+	},
+	"harvestingInterval": {
+	    "type": "object",
+	    "values": {
+		"type": {
+		    "type": "string",
+		    "enum": ["Property"]
+		},
+		"value": {
+		    "$ref": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/common-schema.json/definitions#/definitions/Interval"
+		}
+	    },
+	    "required": ["type", "value"]
+	},
+	"plantingFrom": {
```

Comment:
at this level we don't need to encode all the NGSI-LD structure but just the property name and the value as in other data models